upgrade/1.2/scripts/common/ncn-rebuild-common.sh

```diff
@@ -308,6 +278,9 @@ if [[ $target_ncn != ncn-s* ]]; then
     } >> ${LOG_FILE} 2>&1
 fi 
 
+# force time to sync immediately
+ssh "${target_ncn}" -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null chronyc makestep
```

Comment:
Will that fail if it is not able to synchronize the times? We wouldn't want to proceed if the sync fails.

Reply:
I can add a check here.  This command by itself won't do it.